operations/helm/charts/mimir-distributed/templates/smoke-test/smoke-test-job.yaml

```diff
@@ -10,6 +10,7 @@ metadata:
     {{- toYaml .Values.smoke_test.annotations | nindent 4 }}
     {{- end }}
     "helm.sh/hook": test
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
I think you also need to run `make build-helm-tests` from the root of the repo.

Reply:
Done :)